storage/simulation/cluster.go

```diff
@@ -272,21 +296,24 @@ func (c *Cluster) String() string {
 	return buf.String()
 }
 
-// StringEpochHeader creates the string header for epoch outputs based on all
-// of the current stores.
-func (c *Cluster) StringEpochHeader() string {
-	var buf bytes.Buffer
-	buf.WriteString("Store:\t")
+// OutputEpochHeader outputs to the epoch writer the header for epoch outputs
+// based on all of the current stores.
+func (c *Cluster) OutputEpochHeader() {
+	w := new(tabwriter.Writer)
+	w.Init(c.epochWriter, 0, 8, 0, '\t', 0)
+	fmt.Fprintf(w, "Store:\t")
 	for _, storeID := range c.storeIDs {
-		fmt.Fprintf(&buf, "%d\t", storeID)
+		fmt.Fprintf(w, "%d\t", storeID)
 	}
-	return buf.String()
+	_ = w.Flush()
```

Comment:
I'm not totally sure when this is needed, but perhaps it can be omitted here (and called only once on shutdown).